tony-core/src/main/java/com/linkedin/tony/models/JobMetadata.java

```diff
@@ -142,4 +134,11 @@ public void setId(String id) {
   public void setUser(String user) {
     this.user = user;
   }
-}
+
+  @Override
+  public String toString() {
+    return "\nMetadata{" + "id: " + id + ",\n joblink: " + jobLink + ",\n configlink: " + configLink + ",\n rmLink: "
```

Comment:
I would suggest removing all the `\n`s in the returned string. It's more standard in my experience to keep the default toString() representation a single line and let the user add newlines if they so desire (example: if iterating over a list of JobMetadata, print one JobMetadata per line). Also, when you use `grep`, by default it returns matching lines, so it's generally more convenient when an object's entire string representation is on one line. If a user really wants to print each part on a separate line, they can always call getId(), getJobLink(), etc. separately and construct their own custom string.

You can also merge the first two string segments together: `"Metadata{" + "id: "` --> `"Metadata { "id: "`